src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -4,17 +4,19 @@ sidebar_order: 30
 ---
 <!-- WIZARD -->
 To use Sentry with your React application, you will need to use `@sentry/browser` (Sentry’s browser JavaScript SDK).  
```

Comment:
I would insert the install/configure instructions here, then error boundaries comes after they're fully setup